solr/solrj/src/test/org/apache/solr/client/solrj/io/stream/StreamDecoratorTest.java

```diff
@@ -3048,7 +3048,7 @@ public void testPriorityStream() throws Exception {
       expression =
           StreamExpressionParser.parse(
               "priority(topic(collection1, collection1, q=\"a_s:hello\", fl=\"id,a_i\", id=1000000, initialCheckpoint=0),"
```

Comment:
Looking at what the `testPriorityStream` test is testing, I wonder if the `initialCheckpoint=0` here and at lines 3035/3036 need also be removed?
```suggestion
              "priority(topic(collection1, collection1, q=\"a_s:hello\", fl=\"id,a_i\", id=1000000),"
```

Having said that, it's not very obvious when the checkpoints from the line 3019/3020 stream will be  persisted and/or otherwise carry-over to the line 3035/3036 stream.

Reply:
I wasn't sure whether to remove initialCheckpoint=0, or alter the test expected results which is what I did. Shall I remove the initialCheckpoint=0?